README.md

```diff
@@ -0,0 +1,30 @@
+# Potato Farmer
+
+Manages `potato_field` and `potato_plants`. Main way to administrate containers.
```

Comment:
`potato_field` is the api gateway, so idk if "manages" makes the most sense here. I dont have any alternatives rn though

Reply:
Yeah, I just took this from the repo sidebar but I do feel similarly.